src/main/java/com/epam/prd/jjop/helpers/comparingandobjects/MainComparingObjects.java

```diff
@@ -0,0 +1,115 @@
+package com.epam.prd.jjop.helpers.comparingandobjects;
+
+import java.util.Objects;
+
+/**
+ * I would like to show how to compare objects.
+ *
+ * @author Dymitr Misiejuk
+ */
+class MainComparingObjects {
+    public static void main(String[] args) {
+
+        /*
+          First we need to create objects which are different.
+         */
```

Comment:
```suggestion
          // First, we need to create objects which are different.
```

With those oneliners I would stick to one-line comment syntax